wsrtc-server.mjs

```diff
@@ -210,6 +210,13 @@ const bindServer = (server, {initialRoomState = null, initialRoomNames = []} = [
             })
             break;
           }
+          case MESSAGE.CHAT: {
+            room.players.forEach(player => {
+              if (player === localPlayer) return;

```

Comment:
How about the server doesn't reflect messages back to the sayer?

Reply:
Not sure what you mean, currently it checks if player === localPlayer and doesn't reflect it.

Reply:
Ah yup never mind, I thought this was frontend code!